Makefile

```diff
@@ -138,6 +138,15 @@ libvirt-share: ## Configure ACLs to allow RWX for libvirt VM (e.g. Admin Worksta
 	@find "$(PWD)" -type d -and -user $$USER -exec setfacl -m u:libvirt-qemu:rwx {} +
 	@find "$(PWD)" -type f -and -user $$USER -exec setfacl -m u:libvirt-qemu:rw {} +
 
+.PHONY: vagrant_package
+vagrant_package: ## Package up a vagrant box of the last stable SD release
+	@devops/scripts/vagrant_package.sh
+
+.PHONY: translate
+translate: ## Update POT translation files from sources
+	@cd securedrop ; ./manage.py translate-messages --extract-update
+	@cd securedrop ; ./manage.py translate-desktop --extract-update
```

Comment:
Nit: should probably use `&&` rather than `;` here.

Reply:
ummmmm  i dont remember touching this section :confused: maybe it was a merge-conflict that i fixed while rebasing

Reply:
Good read, thanks for clarifying. Looks like the `translate` Makefile target was introduced in https://github.com/freedomofpress/securedrop/pull/2347, but removed in https://github.com/freedomofpress/securedrop/pull/3087. The `translate` target is currently _absent_ from develop, and should remain so after merge of this PR. Can you amend, @msheiny?